databuilder/file_formats/csv.py

```diff
@@ -79,7 +79,7 @@ def read_dataset_csv(filename, column_specs):
 
 def read_dataset_csv_gz(filename, column_specs):
     with gzip.open(filename, "rt", newline="") as f:
-        return read_dataset_csv_lines(f, column_specs)
+        yield from read_dataset_csv_lines(f, column_specs)
```

Comment:
As before, I don't understand why we use `yield from` now, but we used `return` before. If we should have used `yield from` before, then a rebase would have been helpful.

Reply:
yes, sorry I thought I'd squashed those together in a rebase but clearly not!